firstpartydata/tests/extractfpdforbidders/valid-bidder-fpd-multiple-app-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "inputRequestData": {
+    "id": "bid_id",
+    "site": {
+      "id": "reqSiteId"
+    },
+    "user": {
+      "id": "reqUserId"
+    },
+    "test": 1,
+    "ext": {
+      "prebid": {
+        "data": {
+          "bidders": [
+            "appnexus"
+          ]
+        },
+        "bidderconfig": [
+          {
+            "bidders": [
+              "appnexus"
+            ],
+            "config": {
+              "ortb2": {
+                "site": {
+                  "id": "apnFpdSiteId1"
+                }
+              }
+            }
+          },
+          {
+            "bidders": [
+              "appnexus"
+            ],
+            "config": {
+              "ortb2": {
+                "app": {
+                  "id": "apnFpdAppId2"
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "outputRequestData": {},
+  "biddersFPDResolved": {},
+  "validationErrors": ["duplicated config for bidder: appnexus"]
```

Comment:
`valid-bidder-fpd-multiple-app-config.json` tests is a little different. Yes, there are 2 configs for appnexus and we have a duplicate bidder config error, but it also has config with app and site. This test checks we have duplicate bidder config error, not error about app was not specified in original request. 
Agree, name should be different. But do you think I need to delete it because we have `invalid-fpd-duplicated-bidder-config.json`?